### PR TITLE
Bug 1810748: Added feature gate for CSV Actions in Installed Operators

### PR DESCRIPTION
**Pull request by bipuladh**
- Disabled `Add Capacity` and `Download Cluster Metadata` from Independent Mode storage cluster CR.

### CODE REVIEW

File: frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@ namespace ExtensionProperties {
     apiGroup: string;
     /** action callback */
     callback: (kind: K8sResourceKindReference, obj: any) => () => any;
+    required?: string | string[];
```
Comment (TheRealJon) [inline frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts]: Do we really need to support two different types for this? Can it just be `string[]`, and if it's a single value, the array will just be length 1.

File: frontend/packages/operator-lifecycle-manager/src/components/operand.tsx [hunk 2]
```diff
@@ -245,7 +248,9 @@ export const OperandStatusIconAndText: React.FunctionComponent<OperandStatusIcon
   return iconAndText;
 };
 
-export const OperandTableRow: React.FC<OperandTableRowProps> = ({ obj, index, key, style }) => {
+export const OperandTableRow = connectToFlags<OperandTableRowProps & WithFlagsProps>(
```
Comment (TheRealJon) [inline frontend/packages/operator-lifecycle-manager/src/components/operand.tsx]: Should we use connectToFlags on OperandList and pass the flags through to this component? I think in most cases, where possible, we like to connect higher up and pass the state down to children.

File: frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts (hunk 1 above)
Comment (bipuladh) [reply frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts]: https://github.com/openshift/console/blob/master/frontend/packages/console-plugin-sdk/src/registry.ts#L175-L178
Keeping it consistent with this.

File: frontend/packages/operator-lifecycle-manager/src/components/operand.tsx (hunk 2 above)
Comment (bipuladh) [reply frontend/packages/operator-lifecycle-manager/src/components/operand.tsx]: Wouldn't we have to make changes to Table to configure it to pass the flags props to the row? IMO that would be more complicated than this.

**Comment by spadgett** [comment]
/hold
merge queue is broken

**Pull request merged**